helm/values.yaml

```diff
@@ -0,0 +1,160 @@
+# Default values for metacatui.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+## appConfig contains the MetacatUI.AppConfig settings that can be overridden. Uses a configMap to
+## define MetacatUI.AppConfig, replacing the version of config/config.js found on the mounted drive.
+## Note that changes to the configMap will not be read unless the pod is restarted, and changes to
+## these values will not be reflected in the configMap unless you do a 'helm upgrade'.
+##
+appConfig:
+  ## appConfig.enabled Use a configMap to define MetacatUI.AppConfig, replacing the on-disk version
+  ## Typical use is "false" for development purposes, and "true" for more-permanent deployments
+  ##
+  enabled: true
+
+  ## appConfig.root (required) The url root to be appended after the appConfig.baseUrl, below
+  ##
+  root: "/"
+
+  ## appConfig.theme (required) Corresponds to the name of one of the directories in src/js/themes/
+  ##
+  theme: "default"
+
+  ## appConfig.baseUrl (required) the base url (typically defined by the ingress)
+  ##
+  baseUrl: "http://localhost:8080/"
+
+  ## Optional configuration -- leave commented to use metacatui default values
+  ##
+#  d1CNBaseUrl: "https://cn-sandbox.test.dataone.org/cn"
+#  mapKey: "your-map-key-here"
+#  mdqBaseUrl: "https://localhost:8080:30443/quality"
+#  dataoneSearchUrl: "https://search-stage.test.dataone.org"
+#  portalLimit: 100
+#  googleAnalyticsKey: "your-google-analytics-key-here"
+#  bioportalAPIKey: "your-bio-portal-api-key-here"
+#  cesiumToken: "your-cesium-token-here"
```

Comment:
Can we set any of the appConfig values here?

Reply:
yes - changed comment to reflect this